src/providers/azure.rs

```diff
@@ -557,28 +579,28 @@ impl std::str::FromStr for Region {
             r if r == Region::SouthAfricaNorth.as_ref() => Region::SouthAfricaNorth,
             r if r == Region::UaeNorth.as_ref() => Region::UaeNorth,
             r if r == Region::GermanyWestCentral.as_ref() => Region::GermanyWestCentral,
-            u => bail!("Unknown azure region {}. Valid regions: eastus, eastus2, westus, centralus, northcentralus, southcentralus, northeurope, westeurope, eastasia, southeastasia, japaneast, japanwest, australiaeast, australiasoutheast, australiacentral, brazilsouth, southindia, centralindia, westindia, canadacentral, canadaeast, westus2, westcentralus, uksouth, ukwest, koreacentral, koreasouth, francecentral, southafricanorth, uaenorth, germanywestcentral", u),
+            u => eyre::bail!("Unknown azure region {}. Valid regions: eastus, eastus2, westus, centralus, northcentralus, southcentralus, northeurope, westeurope, eastasia, southeastasia, japaneast, japanwest, australiaeast, australiasoutheast, australiacentral, brazilsouth, southindia, centralindia, westindia, canadacentral, canadaeast, westus2, westcentralus, uksouth, ukwest, koreacentral, koreasouth, francecentral, southafricanorth, uaenorth, germanywestcentral", u),
```

Comment:
I would move the `Valid regions:` bit here into the help text in `color_eyre` via the `Help` trait. I should probably add an `Info` help type for this, right now color-eyre only has categories for suggestions, warnings, and notes.

Reply:
[Like so](https://github.com/jonhoo/tsunami/pull/32/commits/1e0ea8b7815ffb20e41657819445d43b7412437b) I think?

Reply:
precisely